app/src/main/java/com/example/amphibians/ui/screens/HomeScreen.kt

```diff
@@ -131,11 +140,15 @@ fun AmphibianCard(amphibian: Amphibian, modifier: Modifier = Modifier) {
 }
 
 @Composable
-private fun AmphibiansListScreen(amphibians: List<Amphibian>, modifier: Modifier = Modifier) {
+private fun AmphibiansListScreen(
+    amphibians: List<Amphibian>,
+    modifier: Modifier = Modifier,
+    contentPadding: PaddingValues = PaddingValues(0.dp)
+) {
     LazyColumn(
         modifier = modifier,
-        verticalArrangement = Arrangement.spacedBy(24.dp),
-        contentPadding = PaddingValues(dimensionResource(R.dimen.padding_medium))
+        contentPadding = contentPadding,
+        verticalArrangement = Arrangement.spacedBy(24.dp)
```

Comment:
This is not yet addressed. 
`24.dp` is still hard coded. Given all other padding is defined in `dimens.xml`. 
It would in line with the current implantation.